package.json

```diff
@@ -17,6 +17,7 @@
     "dompurify": "1.0.9",
     "esn-frontend-common-libs": "github:openpaas-suite/esn-frontend-common-libs#main",
     "jmap-draft-client": "^0.1.2",
+    "lodash": "^2.4.1",
```

Comment:
really?

Reply:
not sure indeed... let me test it I'll let you know

Reply:
esn-frontend-inbox is broken on `main`...
```
rm -rf node-modules
npm i
npm run serve
```
gives tons of errors like:
```
ERROR in ./src/linagora.esn.unifiedinbox/app/services/selection/selection.service.js
Module not found: Error: Can't resolve 'lodash' in '/home/mbailly/work/openpaas/git/esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/services/selection'
```

I'm fixing this and I'll come back to this PR after